### PR TITLE
feat(livekit): add role

### PR DESCRIPTION
**Pull request by blueorca363**

### CODE REVIEW

File: roles/livekit/README.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+- famedly.base.redis
+
+## Example Playbook
+
+```yaml
+- name: Deploy Livekit Server - an SFU for video calls
+  hosts: [ livekit ]
+  become: true
+  pre_tasks:
```
Comment (sebastian-de) [inline roles/livekit/README.md:98]: should be skipped - see my comment in your local PR.

File: roles/livekit/defaults/main.yml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+livekit_version: "v1.7"
+livekit_jwt_service_version: "sha-4a29504"
+livekit_user: "livekit"
+livekit_container_image_reference: "livekit/livekit-server:{{ livekit_version }}"
+livekit_config_path: "/opt/livekit"
+livekit_config_file: "livekit.yaml"
+livekit_container_config: "/etc/livekit.yaml"
+livekit_container_command: "--config {{ livekit_container_config }}"
+livekit_config_basic:
+  port: 7880
```
Comment (sebastian-de) [inline roles/livekit/defaults/main.yml:11]: As mentioned in the local PR, ideally external ports should be made configurable (at least if it doesn't complicate the configuration too much).

**Review by sebastian-de — CHANGES_REQUESTED** [review]
Apart from that LGTM!

File: roles/livekit/defaults/main.yml [hunk 3]
```diff
@@ -0,0 +1,78 @@
+---
+livekit_version: "v1.7"
+livekit_jwt_service_version: "sha-4a29504"
+livekit_domain: "{{ famedly_instance_domain }}"
+livekit_turnserver_domain: "turn.dev.famedly.de"
```
Comment (sebastian-de) [inline roles/livekit/defaults/main.yml:5]:
```suggestion
livekit_turnserver_domain: ~
```

**Review by sebastian-de — CHANGES_REQUESTED** [review]

File: roles/livekit/defaults/main.yml (hunk 3 above)
Comment (blueorca363) [reply roles/livekit/defaults/main.yml]: Done.

**Review by sebastian-de — CHANGES_REQUESTED** [review]
I'd name the commmit `feat(livekit): add role`

**Review by sebastian-de — APPROVED** [review]

**Pull request merged**